ecs_logging/_stdlib.py

```diff
@@ -112,6 +112,24 @@ def __init__(self, stack_trace_limit=None, exclude_fields=()):
         self._exclude_fields = frozenset(exclude_fields)
         self._stack_trace_limit = stack_trace_limit
 
+    def _record_error_type(self, record):
+        # upstream doesn't handle booleans for `exc_info`
+        if not record.exc_info:
```

Comment:
Can we add comments for the two complex situations within `if` blocks below? Something about what the situation is when this branch is taken and how it's handled

Reply:
Sure, should have removed that comment here too. It is in a subclass that I created to use `ecs_logging` in my project. It has no meaning here.

Pushed an update with more comments.